src/testing/token.js

```diff
@@ -0,0 +1,31 @@
+/* Token generation functions (intended for testing).
```

Comment:
I'm not clear why we want JWT-related functions here. We don't use JWT signing in conjunction with OpenAPI requests.